syntax = "proto3";
package demo;

message Hand{
    repeated float flex_resistor    = 1;   //size 5
    repeated float imu_accel        = 2;      // size 3
    repeated float imu_gyro         = 3;      // size 3
    repeated float imu_mag          = 4;      // size 3
    
}

message Glove{
    repeated float pressure_sensor = 1; //size 5
}
    
    


